src/components/Layout/Header/style.module.less

```diff
@@ -89,6 +89,21 @@
       }
     }
   }
+
+  button {
+    background: none;
+    color: inherit;
+    border: none;
+    padding: 0;
+    font: inherit;
+    cursor: pointer;
+    outline: inherit;
+
+    &:hover {
+      .linkUnderlined();
+      margin-bottom: -0.05em;
```

Comment:
Yup that is a tricky one. I was first wondering why it _is_ working for the links in the menu but not the button. It's because buttons can't become real inline-elements, they will be only inline-block. Dang! https://stackoverflow.com/questions/27656634/button-display-inline-css

For the same reason we got problems with inline buttons that are supposed to look like links, see this PR https://github.com/grundeinkommensbuero/website/pull/135

So I am not sure if it's worth also transforming this into a clickable `<span />` with all the hoops to keep it accessible or if maybe this is an acceptable amount of hackiness.

Reply:
it's a pretty small hack, so i don't mind it. thanks for looking in to the root cause.